resources/views/dashboard/index.blade.php

```diff
@@ -0,0 +1,114 @@
+@extends('layouts.app')
+@section('title', 'Item View')
+@section('content')
+
+<link rel="stylesheet" href="{{ asset('/css/dashboard.css')  }}" >
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.3.0/Chart.bundle.min.js">
+</script>
+
+<div id="dashboard">
+
+  <div class="container">
+
+    <h1>Dashboard</h1>
+
+    <div class="row dashboard-top">
+      <div class="col date-range rounded me-1">
+          <p class="title">Auto date range</p>
+        <div class="form-group">
+          <select class="form-control text-center fw-bold" id="FormSelect1">
+            <option value="" selected>This Month</option>
+            <option value="">All</option>
+            <option value="">All</option>
+            <option value="">All</option>
+          </select>
+        </div>
+      </div>
+      <div class="col service rounded mx-1">
+        <p class="title">Services</p>
+        <div class="form-group">
+          <select class="form-control text-center fw-bold" id="FormSelect2">
+            <option value="" selected>All</option>
+            <option value="">All</option>
+            <option value="">All</option>
+            <option value="">All</option>
+          </select>
+        </div>
+      </div>
+      <div class="col post rounded ms-1">
+        <p class="title">Post</p>
+        <div class="form-group">
+          <select class="form-control text-center fw-bold" id="FormSelect3">
+            <option value="" selected>All</option>
+            <option value="">All</option>
```

Comment:
is this a temporarily or just permanent values?

Reply:
This is a temporaly values.